tests/appsec/api_security/test_api_security_rc.py

```diff
@@ -0,0 +1,109 @@
+# Unless explicitly stated otherwise all files in this repository are licensed under the the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2021 Datadog, Inc.
+
+from utils import (
+    context,
+    coverage,
```

Comment:
I've just removed this in #2091, you can remove the import, and the later references

Reply:
Thank you for the fix !

Reply:
(sorry for the trouble 😓 )